workflow-templates/deploy-application.yml

```
@@ -0,0 +1,159 @@
+# Workflow to deploy to an AWS environment with maven, terraform, build-image
+# This workflow requires the maven wrapper to be in the root of the project
+
+name: 'Deploy to AWS'
+
+on:
+  push:
+    branches:
+      - $default-branch
+
+concurrency:
+  group: ${{ github.workflow }}
+
+env:
+  AWS_REGION: 'eu-west-1'
+  AWS_ROLE: '' # TODO Fill in the role to assume for your deployment
+  TF_DIR_INFRA: '' # TODO Fill in the terraform directory for your infra deployment
+  TF_DIR_HELM: '' # TODO Fill in the terraform directory for your helm deployment
+  JAVA_VERSION: '17'
+
+jobs:
+
+  deploy_infra:
+    needs: release_please
+    name: "Deploy infra"
```

```suggestion
    name: Deploy infra
```

Geen idee of het nodig is, maar zie het nergens anders